### PR TITLE
First draft of collaboration section

### PR DESCRIPTION
**Pull request by lexnederbragt**
While I was in writing mode I gave a shot at the collaboration too.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -178,36 +178,33 @@ FIXME
...
-    are finding a task to start on
-    and problems setting up the local workspace to start work
-    *   So remove those
-3.  Additional disincentive is uncertainty: what am I allowed to do?
-    *   So remove that as well
-    *   Lack of an explicit license implies the author is keeping all rights
-        and others are not allowed to re-use
+You may work on your project with others, your known collaborators. But, even at an early stage, you may be interested to open your project so that you can attract new, as yet unknown collaborators. This section list approaches for making it straightforward for others to start collaborating on your project.
+
+1.  *Simplicity:* the easier it is for people to collaborate, the more likely they are to do so (and to give you credit)
```
Comment (gvwilson) [inline index.md:183]: These aren't approaches, they're reasons - add an introductory sentence before the list to say that?

File: index.md [hunk 2]
```diff
@@ -178,36 +178,33 @@ FIXME
...
 ### Rules
 
-1.  Every project has a short README file explaining its purpose
-    *   Includes a contact address that actually works
-2.  And a plain text file (often called notes.txt or todo.txt) containing the to-do list
-    *   Aimed at contributors, where README is aimed at users
-3.  And a LICENSE file
+1.  Have a short README file explaining the project's purpose.  
+This file should includes contact information that actually works. This file is often the first thing users of your project will look at, so make it explicit already here that you welcome contributors and point them to the ways to help out. Consider using a CONTRIBUTION file to describe any steps needed to start working: dependencies that need to be installed, running of (unit) tests, guidelines or rules that your project adheres to (e.g. on commit messages or checklists you may use before accepting a suggested change).
+2.  Have a plain text file containing the to-do list.
```
Comment (gvwilson) [inline index.md]: Rephrase this as "have a shared public to-do list", and then talk about using a text file vs. GitHub issues as implementations?

File: index.md [hunk 3]
```diff
@@ -178,36 +178,33 @@ FIXME
...
+3.  Have a LICENSE file.  
+Lack of an explicit license implies the author is keeping all rights and others are not allowed to re-use or modify the material. We recommend:
     *   CC-0 or CC-BY for data and text
-    *   Permissive license (e.g., MIT/BSD/Apache) for software
-4.  And a CITATION file
+    *   A permissive license (e.g., MIT/BSD/Apache) for software
+4.  Have a CITATION file.
+This file describes:
     *   How to cite this project overall
     *   Where to find/how to cite data sets, code, figures, and other things that have their own DOIs
```
Comment (gvwilson) [inline index.md:202]: An example of a CITATION file would be helpful.

File: index.md [hunk 4]
```diff
@@ -178,36 +178,33 @@ FIXME
...
+1.  Have a short README file explaining the project's purpose.  
+This file should includes contact information that actually works. This file is often the first thing users of your project will look at, so make it explicit already here that you welcome contributors and point them to the ways to help out. Consider using a CONTRIBUTION file to describe any steps needed to start working: dependencies that need to be installed, running of (unit) tests, guidelines or rules that your project adheres to (e.g. on commit messages or checklists you may use before accepting a suggested change).
+2.  Have a plain text file containing the to-do list.  
+Call this file `notes.txt` or `todo.txt` or something similar, and make descriptions of the items clear enough for new collaborators. On Github, you could use the *issue* functionality, and create a new issue for each to-do item (you can even add labels such as 'low hanging fruit' to point people to where to get started).
+3.  Have a LICENSE file.  
+Lack of an explicit license implies the author is keeping all rights and others are not allowed to re-use or modify the material. We recommend:
     *   CC-0 or CC-BY for data and text
```
Comment (gvwilson) [inline index.md]: We'll need to explain the acronyms CC-0 and CC-BY.

**Comment by gvwilson** [comment]
ETA for merge?

**Comment by lexnederbragt** [comment]
Tomorrow, most likely
